src/firefly/js/ui/UploadTableSelector.jsx

```diff
@@ -373,8 +376,8 @@ export function CenterColumns({lonCol,latCol, sx, cols, lonKey, latKey, openKey,
                                   doQuoteNonAlphanumeric, headerTitle='Position Columns:',
                                   headerPostTitle = '', posDefaultOpenMsg='', setPosDefaultOpenMsg, slotProps}) {
     const columnFields = positionColumnFields(
-        {fieldKey: lonKey, doQuoteNonAlphanumeric},
-        {fieldKey: latKey, doQuoteNonAlphanumeric}
+        {fieldKey: lonKey, doQuoteNonAlphanumeric, colTblId: 'posCol', onSearchBtnClicked: () => filterMappedColFldTbl(cols, 'posCol', 'pos.eq.ra,pos.galactic.lon,pos.ecliptic.lon')},
+        {fieldKey: latKey, doQuoteNonAlphanumeric, colTblId: 'posCol', onSearchBtnClicked: () => filterMappedColFldTbl(cols, 'posCol', 'pos.eq.dec,pos.galactic.lat,pos.ecliptic.lat')}
```

Comment:
you need to support `pos.eq`. This is when the ra and dec are in the same column.
test on TAP with CANFAR ->  pgm -> gaiadr2